src/models/device.ts

```diff
@@ -1261,17 +1262,64 @@ const getDeviceModel = function (
 		 * });
 		 *
 		 * @example
+		 * balena.models.device.getSupervisorTargetState(123, 3).then(function(state) {
+		 * 	console.log(state);
+		 * });
+		 *
+		 * @example
 		 * balena.models.device.getSupervisorTargetState('7cf02a6', function(error, state) {
 		 * 	if (error) throw error;
 		 * 	console.log(state);
 		 * });
 		 */
 		getSupervisorTargetState: async (
 			uuidOrId: string | number,
+			version: 2 | 3 = 2,
```

Comment:
Doesn't this need an overload in order to have the correct return type when using v3?